mavlink_vehicles/mavlink_vehicles.cc

```diff
@@ -41,6 +41,38 @@
 #define print_verbose(...) ;
 #endif
 
+enum PX4_CUSTOM_MAIN_MODE {
```

Comment:
I'm not sure if this is the best place to keep these definitions. Qgroundcontrol keeps PX4 and APM definitions in separate files into a firmware folder. Should we do something similar? @rchiossi